src/commands/init.rs

```diff
@@ -77,10 +82,87 @@ complete -F _{function_name}_completions {function_name}
     };
 }
 
-// given the location of the cookbook executable, return
-// back the init script for cookbook.
+macro_rules! fish_init_body {
```

Comment:
It might be nice to replace these macro bodies with something like [include_str](https://doc.rust-lang.org/std/macro.include_str.html).

Reply:
Ok, want that tweak in this PR or follow on issue?

Reply:
follow on is fine, non-blocking comment.

Reply:
https://github.com/toumorokoshi/tome/issues/11